Persephone/MPDClient/Models/Song.swift

```diff
@@ -41,6 +41,10 @@ extension MPDClient {
       mpd_song_free(mpdSong)
     }
 
+    var uri: UnsafePointer<Int8> {
```

Comment:
I guess this is a C string? Would it make sense to wrap this in a `String` at this level?

Reply:
Actually, in the place I'm using this I need it to be the `UnsafePointer` because I'm passing it to a `libmpdclient` method.